src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1585,6 +1597,104 @@ public ReportType updateCounts( final CodonTracker codonTracker,
             return ReportType.CALLED_VARIANT;
         }
 
+        // try to find a supplemental alignment adjacent to the primary alignment on the read
+        // that is sensible (same strand, same order) and disjoint on the reference.
+        // if there is such a supplemental alignment, combine the primary and supplemental alignments,
+        // adding a large deletion in the middle to represent the missing reference bits.
+        @VisibleForTesting static Cigar findLargeDeletions( final GATKRead read ) {
+            Cigar cigar = read.getCigar();
+            final List<CigarElement> elements = cigar.getCigarElements();
+            final int nElements = elements.size();
+            if ( nElements < 2 ) return cigar;
+            final int initialClipLength = getClipLength(elements.get(0));
+            final int finalClipLength = getClipLength(elements.get(nElements - 1));
+            if ( initialClipLength >= minAltLength || finalClipLength >= minAltLength ) {
```

Comment:
because that's the part of the read that could support a supplemental alignment (which must be mostly non-overlapping with the primary alignment, or else it would be marked secondary instead).

Reply:
is it guaranteed to be this way?
Judged by 1646, the overlap (or gap) between the two alignment records on the query itself is practically limited to 1 or 2. But it is imaginable that the micro-homology flanking the large deletion (or micro insertions accompanying the deletion) might be larger than that, in which case this method might be loosing some sensitivity.

Reply:
These are "by design" deletions, not natural mutations.  We expect them to be clean.